src/scheduler/pbs_sched_utils.cpp

```diff
@@ -588,15 +597,15 @@ connect_svrpool()
 				/* wait for 2s for not to burn too much CPU, and then retry connection */
 				sleep(2);
 				close_servers();
-				continue;
+				goto CONTINUE;
```

Comment:
How about moving the sleep(2) and close_servers() to the CONTINUE label, and unblocking the signals before going to sleep, so that scheduler doesn't block them while it's sleeping